recipes/nextsilicon-cpp-subprocess/all/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, tools
+
+
+class CppSubprocess(ConanFile):
+    name = "nextsilicon-cpp-subprocess"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nextsilicon/cpp-subprocess#readme"
+    topics = ("subprocess", "os", "fork")
+    description = ("Subprocessing with modern C++, "
+                   "The only goal was to develop something that is as close as"
+                   "python subprocess module in dealing with processes.")
+    no_copy_source = True
```

Comment:
```suggestion
    no_copy_source = True
    settings = "os", "arch", "compiler", build_type"
```

I think conan v2 requires this, and it doesn't hurt since there is `self.info.header_only()` in `package_id()`